global_helpers/panther_greynoise_helpers.py

```diff
@@ -54,6 +54,13 @@ def actor(self, match_field) -> str:
     def url(self, match_field) -> str:
         return f"https://www.greynoise.io/viz/ip/{deep_get(self.noise, match_field, 'ip')}"
 
+    def context(self, match_field) -> dict:
```

Comment:
handy!

Reply:
Right? Thank you,  it came to me while doing something completely unrelated over the weekend.  Brains are weird.